pkg/kubelet/kubelet.go

```diff
@@ -1554,6 +1555,10 @@ func (kl *Kubelet) syncLoop(updates <-chan PodUpdate, handler SyncHandler) {
 			unsyncedPod = true
 		case <-time.After(kl.resyncInterval):
 			glog.V(4).Infof("Periodic sync")
+		case <-syncStatusTicker:
+			glog.V(4).Infof("Periodic status sync")
+			kl.syncStatus()
+			continue
```

Comment:
I see your point, but I don't think that removing continue is the way to go. This assumes how often syncStatusTicker ticks. If we change it to 0.1 * kl.resyncInterval, it would be syncing pods too often, right?

How about we just have two tickers - one for syncing pods and one for syncing status. We will effectively do a bit more pods syncing, but I guess it should be acceptable and it will also make the code more readable (currently it's kind of hard to understand for me when something is triggered). WDYT?